config/zluxserver.json

```diff
@@ -54,8 +55,10 @@
       },
       "zss": {
         "plugins": ["org.zowe.zlux.auth.zss"]
-      } 
-      
+      },
+      "apiml": {
+        "plugins": ["org.zowe.zlux.auth.apiml"]
```

Comment:
If we commit this one line, it actually makes this repo require APIML to be running in order to log into the desktop, since the desktop multicasts credentials to all auth plugins, and will fail if any one fails. So we'd need to update all docs to link to a doc on building & configuring APIML. Should we instead leave this out of the source code, and instead be inserting this in the installer, since we know the installer comes with APIML?

Reply:
I have commented this out.